hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/prehandle/PreHandleWorkflowImplTest.java

```diff
@@ -147,7 +145,8 @@ void setUp() {
     /** Null arguments are not permitted to the constructor. */
     @Test
     @DisplayName("Null constructor args throw NPE")
-    @SuppressWarnings("DataFlowIssue") // Suppress the warning about null args
+    @SuppressWarnings("DataFlowIssue")
```

Comment:
Putting the comment in a new line makes it less clear what it is referring to. If Spotless forces us to put the comment in a new line, it should be above the relevant code.